server/game/cards/15-W2D/AsNatureAndGodIntended.js

```diff
@@ -0,0 +1,43 @@
+const ActionCard = require('../../actioncard.js');
+
+class AsNatureAndGodIntended extends ActionCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'As Nature and God Intended',
+            playType: ['shootout'],
+            message: context =>
+                this.game.addMessage(' {0} uses {1} to disable all Weapons and make all Kung Fu dudes studs until end of this round', context.player, this), // TODO: Not sure about what to output here, regurgitating the card's effect is too verbose. [ST 2021/07/07]
```

Comment:
You can delete the TODO and I know I suggested it, but I believe the message should end: "... until end of this shootout round" to make clear it does not last until the end of the day (round).

Reply:
:heavy_check_mark: